### PR TITLE
Moved blocks and fix to resman for FAST v33-v34 transition

### PR DESCRIPTION
**Pull request by ludoo**
This is a followup from #2512 and a simple way we should provide guidance on updating FAST for every release. The fast-test-05 org is pinned to last version, and running apply on bootstrap, resman, and net-a allows us to write moved blocks and note down major changes.

The release notes for v34 should add a section

### Bootstrap stage

No destructive changes. A few IAM bindings are reapplied cleanly.

### Resource management stage

Network security IaC resources change names from `resman-netsec` to `resman-nsec` and need recreation. Network security state should be transitioned to local before applying resource management, and re-transitioned to remote after refreshing resman output files and netsec provider.

Project factory dev and prod resources will change internal names, the moved blocks in the provided file should seamlessly rename them in state.

### CODE REVIEW

File: fast/stages/1-resman/branch-nsec.tf [hunk 1]
```diff
@@ -16,13 +16,25 @@
 
 # tfdoc:file:description Network security stage resources.
 
-# automation service account
+# TODO: remove in v35.0.0
 
 moved {
```
Comment (sruffilli) [inline fast/stages/1-resman/branch-nsec.tf:21]: Why aren't those in a `moved` file?

Comment (ludoo) [reply fast/stages/1-resman/branch-nsec.tf:21]: because they are not a delta between versions

File: fast/stages/1-resman/v33.0.0-v34.0.0.tf
Comment (juliocc) [inline fast/stages/1-resman/v33.0.0-v34.0.0.tf:1]: why the symlink?

**Review by juliocc — APPROVED** [review]

File: fast/stages/1-resman/v33.0.0-v34.0.0.tf
Comment (ludoo) [reply fast/stages/1-resman/v33.0.0-v34.0.0.tf:1]: there's a symlink? :)

Comment (ludoo) [reply fast/stages/1-resman/v33.0.0-v34.0.0.tf:1]: good catch

**Pull request merged**